### DISCUSSION TITLE
not working on Apple mini M1

### DISCUSSION THREAD

**Discussion by hitchio** [apohl79/audiogridder#631] [General]
although version 1.1.1 (64) is installed, cubase 11.0.30 on apple mini M1 blacklists the plugin as a 32-bit plugin???
therefore i cannot use it.
any ideas?
<img width="709" alt="Bildschirmfoto 2021-07-01 um 21 30 20" src="https://user-images.githubusercontent.com/1135467/124348648-613b3600-dbeb-11eb-8c5f-c8842d020ce7.png">
<img width="288" alt="Bildschirmfoto 2021-07-01 um 21 20 03" src="https://user-images.githubusercontent.com/1135467/124348649-6304f980-dbeb-11eb-8ae2-eb7df91f9a94.png">

**Comment by apohl79** [comment]
You can try the universal binaries, which include intel and arm binaries in one file. Your DAW and the plugins have to be build for the same target architecture (arm or intel).
